modules/database/db_sql_live.cpp

```diff
@@ -136,7 +136,7 @@ class DBMySQL : public Module, public Pipe
 
 	void OnRestart() override
 	{
-		init = false;
+		this->OnRestart();
```

Comment:
This will cause an infinite loop. Did you mean to call another method here?

Reply:
> This will cause an infinite loop. Did you mean to call another method here?

@SadieCat yeah alright, fix it.
https://github.com/anope/anope/pull/295/commits/4fb0a3908ddd4305f06db76d8e60330a92e4a28e